### PR TITLE
Operationalize implementation of predictions on issues

### PR DESCRIPTION
**Pull request by harshp8l**
Since the models are being hosted on a production server, this is meant to operationalize the predictions served by the model by the bot (i.e. take requests from it)

### CODE REVIEW

File: mxnet-bot/LabelBotFullFunctionality/LabelBot.py [hunk 1]
```diff
@@ -303,6 +358,11 @@ def parse_webhook_data(self, event):
                     logging.error(f'Unsupported actions: {actions}')
                     raise Exception("Unrecognized/Infeasible label action for the mxnet-label-bot")
 
+        # On creation of a new issue, automatically trigger the bot to recommend labels
+        if github_event == "issues" and payload["action"] == "opened":
+            self._find_all_labels()
```
Comment (jlcontreras) [inline mxnet-bot/LabelBotFullFunctionality/LabelBot.py]: Why do you call _find_all_labels here? Doesn't seem to be used in the call to predict_label below

Comment (harshp8l) [reply mxnet-bot/LabelBotFullFunctionality/LabelBot.py]: predict_label makes a call to add_label and add_label requires that we find all labels first (i.e. the assert statement initially in add_label)

Comment (jlcontreras) [reply mxnet-bot/LabelBotFullFunctionality/LabelBot.py]: Or maybe even have it in add_label if it is that is the function that requires it? Your call though

Comment (harshp8l) [reply mxnet-bot/LabelBotFullFunctionality/LabelBot.py]: Following the previous convention of find_all_labels in parse_webhook_data. I think its best we stick with making the call here

**Pull request merged**